### PR TITLE
Update Regex.xml to clarify what the static method means in Regex.CacheSize

### PR DESCRIPTION
**Pull request by SanderSade**
## Summary

Clarified what the static method means in the context of Regex.CacheSize. Current documentation is ambiguous, making it look like the discussion is about static C# methods and not Regex,Match/IsMatch/Replace/etc methods.

### CODE REVIEW

**Comment by SanderSade** [comment]
Gentle ping on @dotnet/area-system-text-regularexpressions members.

File: xml/System.Text.RegularExpressions/Regex.xml [hunk 1]
```diff
@@ -676,7 +676,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Text.RegularExpressions.Regex> class maintains an internal cache of compiled regular expressions used in static method calls. If the value specified in a set operation is less than the current cache size, cache entries are discarded until the cache size is equal to the specified value.  
+ The <xref:System.Text.RegularExpressions.Regex> class maintains an internal cache of compiled regular expressions used in static Regex method calls, such as <xref:System.Text.RegularExpressions.Regex.Match%2A> or <xref:System.Text.RegularExpressions.Regex.Replace%2A>. If the value specified in a set operation is less than the current cache size, cache entries are discarded until the cache size is equal to the specified value.
```
Comment (buyaa-n) [inline xml/System.Text.RegularExpressions/Regex.xml:679]:
```suggestion
 The <xref:System.Text.RegularExpressions.Regex> class maintains an internal cache of compiled regular expressions used in static <xref:System.Text.RegularExpressions.Regex> method calls, such as <xref:System.Text.RegularExpressions.Regex.Match%2A> or <xref:System.Text.RegularExpressions.Regex.Replace%2A>. If the value specified in a set operation is less than the current cache size, cache entries are discarded until the cache size is equal to the specified value.  
```

File: xml/System.Text.RegularExpressions/Regex.xml [hunk 2]
```diff
@@ -523,11 +523,11 @@
...
-          <format type="text/markdown"><![CDATA[
-
-## Remarks
- The <xref:System.Text.RegularExpressions.Regex> class maintains an internal cache of compiled regular expressions used in static method calls. If the value specified in a set operation is less than the current cache size, cache entries are discarded until the cache size is equal to the specified value.
-
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The <xref:System.Text.RegularExpressions.Regex> class maintains an internal cache of compiled regular expressions used in static <xref:System.Text.RegularExpressions.Regex> method calls, such as <xref:System.Text.RegularExpressions.Regex.Match%2A> or <xref:System.Text.RegularExpressions.Regex.Replace%2A>. If the value specified in a set operation is less than the current cache size, cache entries are discarded until the cache size is equal to the specified value.  
+
```
Comment (buyaa-n) [inline xml/System.Text.RegularExpressions/Regex.xml:530]:
```suggestion
          <format type="text/markdown"><![CDATA[

## Remarks
 The <xref:System.Text.RegularExpressions.Regex> class maintains an internal cache of compiled regular expressions used in static <xref:System.Text.RegularExpressions.Regex> method calls, such as <xref:System.Text.RegularExpressions.Regex.Match%2A> or <xref:System.Text.RegularExpressions.Regex.Replace%2A>. If the value specified in a set operation is less than the current cache size, cache entries are discarded until the cache size is equal to the specified value.

```

**Review by buyaa-n — APPROVED** [review]
Looks good, thank you

**Pull request merged**